charts/nexus-iq/values.yaml

```diff
@@ -17,6 +17,12 @@ imagePullSecrets:
 # kubernetes secret
 #   - name: <pull-secret-name>
 
+# Change initContainer enabled to true, to change file ownership to new UID, when upgrading from old IQ version (<120, <100)
+initContainers:
```

Comment:
Feel free to ignore as very minor but ideally should not this be some logical name? (eg: "updateOwner")